lib/container_orchestrator/object_definition.rb

```diff
@@ -100,6 +100,11 @@ def deployment_definition(name)
             ],
           }
         }
+
+        if ENV["MESSAGING_KEYSTORE_PASSWORD"].present?
+          volume = deployment[:spec][:template][:spec][:volumes].find { |vol| vol[:name] == "internal-root-certificate" }
+          volume[:secret][:items].append({:key => "kafka_keystore", :path => "kafka.keystore.jks"})
+        end
```

Comment:
Minor (don't bother changing for this PR we can try in a follow-up) but since we're building the `volume` hash above, we should be able to add this in directly rather than try to find it later.  Just seems more direct.